Libs/MRML/Core/vtkITKTransformConverter.h

```diff
@@ -312,13 +312,13 @@ bool vtkITKTransformConverter::SetITKLinearTransformFromVTK(vtkObject* loggerObj
   }
 
   AffineTransformType::Pointer affine = AffineTransformType::New();
-  affine->SetMatrix(itkmat);
-  affine->SetOffset(itkoffset);
   if (center_RAS)
   {
     double center_LPS[3] = { -center_RAS[0], -center_RAS[1], center_RAS[2] };
     affine->SetCenter(center_LPS);
   }
+  affine->SetMatrix(itkmat);
+  affine->SetOffset(itkoffset);
```

Comment:
This [note in the ITK API documentation](https://github.com/InsightSoftwareConsortium/ITK/blob/4f675a2f24f496de0f58dcb17c4e80dc2fb462e8/Modules/Core/Transform/include/itkMatrixOffsetTransformBase.h#L246-L247) suggests that we should set matrix+center+translation OR matrix+offset. Here we set center, matrix, and offset. We should at least explain here in a comment why our implementation is appropriate.

Reply:
I've tried using matrix+center+translation, however that seems to break compatibility with older versions of Slicer. I've added a comment for now explaining why we use center+matrix+offset.

Reply:
I see, we just need to document this well.

Do you know why exactly there is a backward compatibility issue? 

Are older versions of Slicer ignore some important component of the transform?

Are the transforms that we save now loaded correctly in older Slicer versions (it is OK that center of rotation is missing, but is the 4x4 matrix the same)? Is it possible that earlier Slicer versions all loaded certain ITK linear transforms incorrectly?

If we resample an image using an ITK-based CLI with an ITK transform, do we get the same results as we would get if we apply the transform in Slicer?